lms/djangoapps/course_api/blocks/api.py

```diff
@@ -102,15 +103,16 @@ def get_blocks(
         )
     ]
 
-    if include_completion:
-        transformers += [BlockCompletionTransformer()]
```

Comment:
This change is unrelated to my actual purpose.  I just noticed that we were manually reproducing the work of a kwarg to `get_course_blocks` - so figured I'd not duplicate the code.